python/src/otel/otel_sdk/requirements.txt

```diff
@@ -1,3 +1,5 @@
-opentelemetry-exporter-otlp-proto-http==1.19.0
-opentelemetry-distro==0.40b0
-opentelemetry-instrumentation-aws-lambda==0.40b0
+opentelemetry-exporter-otlp==1.20.0
+opentelemetry-exporter-otlp-proto-http==1.20.0
+opentelemetry-exporter-otlp-proto-grpc==1.20.0
+opentelemetry-distro==0.41b0
```

Comment:
Do we need opentelemetry-distro? I was thinking solarwinds-apm itself is a distro

Reply:
Nope not needed. Created a test layer-python to be sure, and works without it. I think this dep is leftover from when I tried to build layer-python as vanilla distro, without APM Python.